### ISSUE TITLE
Ideas I have which needs to be implemented in the beginning

### ISSUE THREAD

**Issue by arihantparsoya** [parsoyaarihant/vision#2]
First thing to implement should be to detect the real height and width of a person from the image given that we know the distance of the person from the camera. the program should be able to take following inputs:
-  front_image, side_image ::   2 image of the person.(Front and the side view. I feel that it will make the process easier if the background is a plane colour) 
- distance :: the distance of the person from the camera in meters/centimertes.
- height :: the height of the camera from the ground level.

Once we have done this we shlould be able to get true dimention of a person even if we change the distance and height variables later.

Lets do things step by step, once this is done then we can start working on 
- creating 3D model of the person.
- Fit the clothes on 3D model.

**Comment by rahulraj1255** [comment]
I have some considerations : 
1) Is the scaling factor for the center of the image same as that for the sides 
2) Background should not be an issue as once we recognize the contour of the person we can easily extract it and do whatever we want ( for eg: draw it on a white image)
3) We can generalize it ( no need to give input of distance from the camera ):
a)We can keep an object of known length by the side of the person for eg a 1 meter scale (less preferred )
b)We can draw lines on the ground which will help in calculating the scaling factor

**Comment by arihantparsoya** [comment]
yeah, since morning I was trying to find an algirithm to find the real size of the person and:
- I found that the reliable method to do that will be to keep a reference object of known size near the image.
- yeah there will be some error with the scaling in the side of image (i dont know how much, i will ask seniors about it and comment it here).
- For now lets just take one photo with plane background(we can change our code later for any background too since thats not the main aim of the project), making it work for any background will need extra work. 

Did u clone this repository?? try running the `test.py` file ( thats my previous project in openCV ) and see if it works in your computer. Note that the function `pupildetector()` is imported from other file. We will be making different function in different files so its easier to work with.

**Comment by rahulraj1255** [comment]
Yeah for the second point I too agree that we can first take plane background and after our project is done we can think of extending it and adding something
Yeah already cloned and executed it on my comp. ( I have a query : For one of the image it fails to detect the pupil and detects some arbit shape as circle so can you explain how for that shape sqrt{A/pi}==P/2pi )

**Comment by arihantparsoya** [comment]
The algorithm was to 
- sort the contours based on the area ( decending order )
- for the top 4 contours calculate the radius based on perimeter ( `perimeter / 2 * pi`) and based on area (`sqrt ( area / pi )` )  and compare the radius found by the two. if the difference between the radius is more, than the contour is less likely to be a circle. 
- the one with less error is taken to be the final answer.
- please update your branch from the my master branch, I took photos from internet and tried to detect the topoist and the bottommost point in the image so we can find the height.

Sorry for commenting late, i didnt recieve notification for your comment.
